airbyte-webapp/.storybook/withProvider.tsx

```diff
@@ -38,11 +39,13 @@ export const withProviders = (getStory) => (
           <MemoryRouter>
             <IntlProvider messages={messages} locale={"en"}>
               <ThemeProvider theme={theme}>
-                <ConfigServiceProvider defaultConfig={defaultConfig} providers={[]}>
+                <ConfigServiceProvider config={config}>
                   <DocumentationPanelProvider>
-                    <FeatureService features={[]}>
-                      {getStory()}
-                    </FeatureService>
+                    <AppMonitoringServiceProvider>
```

Comment:
:information_source: This broke the `ConnectorForm` which I think didn't work on `master` anymore.